packages/ui-library/src/components/feedback/tooltip/index.stories.ts

```diff
@@ -1,123 +1,306 @@
+      table: {
+        category: 'Content / Settings',
+      },
     },
   },
+
   parameters: {
+    viewMode: 'docs',
+    layout: 'padded',
     design: {
       type: 'figma',
       url: 'https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?node-id=3618%3A126744&mode=dev',
     },
+    docs: {
+      description: {
+        component: `
+  <Markdown>
+  Tooltip is triggered by hovering the mouse cursor over the element. It is designed to be unobtrusive and usually consist
+  of a short snippet of text. Tooltip component utilizes Floating UI library for granular positioning and interactions
+  based on user events.
+  
+  - [**Appearance**](#appearance)
+    - [**Placement**](#placement)
+    - [**Elevation**](#elevation)  
+    - [**Has Arrow**](#has-arrow)
+    - [**Offset**](#offset)
+  - [**Content / Settings**](#content--settings)
+    - [**Message**](#message)
+  - [**Content / Settings**](#content--settings)
+    - [**Message**](#message)
+  - [**Dependencies**](#dependencies)
+    - [**Tooltip Bubble**](#tooltip-bubble)
+  </Markdown>
+`,
+      },
+    },
   },
 };
 
-export const BlrTooltip = (params: BlrTooltipType) => html`<div
-  style="height: 400px; width: 400px; display: flex; justify-content: center; align-items: center; margin: auto;"
+export const Tooltip = (params: BlrTooltipType) => html`<div
+  style="height: 300px;  margin: 1.25rem; width: 300px; display: flex; justify-content: center; align-items: center; margin: auto;"
 >
   ${BlrTooltipRenderFunction(
     params,
-    html`<div style="height: 200px; width: 200px; background-color: lightblue"></div>`
+    html`<div style="height: 100px; width: 100px; background-color: lightblue"></div>`
   )}
 </div>`;
 
-export const BlrTooltipVirtualReference = (params: BlrTooltipType) => {
-  return html` <div style="height: 400px; width: 400px">
-    <virtual-reference
-      theme=${params.theme}
-      text=${params.message}
-      placement=${params.placement}
-      hasArrow=${params.hasArrow}
-      elevation=${params.elevation}
-      offset=${params.offset}
-    ></virtual-reference>
-  </div>`;
-};
-
-BlrTooltip.storyName = 'Tooltip';
-BlrTooltipVirtualReference.storyName = 'Tooltip Virtual Reference';
-
-const args: BlrTooltipType = {
+const defaultParams: BlrTooltipType = {
   theme: 'Light',
-  message: 'Tooltip text comes here Tooltip text comes here',
   placement: 'top',
-  hasArrow: true,
   elevation: true,
   offset: 4,
+  hasArrow: true,
+  message: 'Tooltip text comes here Tooltip text comes here',
+};
+
+Tooltip.args = defaultParams;
+
+/**
+ * ## Appearance
+ * ### Placement
+ * The Tooltip component can be positioned in any of the following placements: top, top-start, top-end, right, right-start, right-end, bottom, bottom-start, bottom-end, left, left-start, and left-end.
+ */
+
+export const Placement = () => {
+  return html`
+    ${sharedStyles}
+
+    <div class="wrapper">
+      ${Tooltip({
```

Comment:
I hope you don't mind, but I updated this PR with the latest from `develop`. I think the only recommendation I would have here would be to maybe add a label to each of these examples. On first glance it can be hard to tell which blue box corresponds to which tooltip placement. So maybe add 'a Top Placement' before we call the Tooltip function in side "wrapper" and do some basic styling to make it clear what each example does

Reply:
Yes, good point. I will implement that in my next commit